### ISSUE TITLE
MacOS Error When Trying to Use Layer Diffuse

### ISSUE THREAD

**Issue by booraddley** [LykosAI/ComfyUI-Inference-Core-Nodes#14]
I'm on a MacBook Pro 2021, 16inch, Apple M1 Max, 64GB RAM, 1TB drive ,running MacOS 14.4.1 

I get the following failed assertion when I try to generate an image with in Stability Matrix in Inference, using Layer Diffuse:

`got prompt
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
clip missing: ['clip_l.logit_scale', 'clip_l.transformer.text_projection.weight']
Requested to load SDXLClipModel
Loading 1 new model
Requested to load SDXL
Loading 1 new model
  0%|          | 0/30 [00:00<?, ?it/s]/Users/redacted/Library/Application Support/StabilityMatrix/Packages/ComfyUI/venv/lib/python3.10/site-packages/torchsde/_brownian/brownian_interval.py:608: UserWarning: Should have tb<=t1 but got tb=14.614640235900879 and t1=14.61464.
  warnings.warn(f"Should have {tb_name}<=t1 but got {tb_name}={tb} and t1={self._end}.")
100%|██████████| 30/30 [01:02<00:00,  2.10s/it]
Requested to load AutoencoderKL
Loading 1 new model
100%|██████████| 8/8 [00:05<00:00,  1.53it/s]
/AppleInternal/Library/BuildRoots/ce725a5f-c761-11ee-a4ec-b6ef2fd8d87b/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShaders/MPSNDArray/Kernels/MPSNDArraySort.mm:287: failed assertion `(null)" Axis = 4. This class only supports axis = 0, 1, 2, 3
'`

I googled the crap out of this error and I've got nothing. I assume the crux of the problem is that `brownian_interval.py` didn't load. And then of course `Axis = 4. This class only supports axis = 0, 1, 2, 3` is maybe even worse?  Anyway, I just wanted to report this cause I didn't see it in the list of issues.
